src/styles/variables.scss

```diff
@@ -0,0 +1,17 @@
+$day-width: 100px;
+$day-height: 100px;
+$days-gap-size: 1;
+$days-in-row: 7;
+$calendar-paddings: 10px;
+$calendar-width: (
+  $day-width + $days-gap-size ) * $days-in-row - $days-gap-size;
+$days: (
+  mon 1,
+  tue 2,
+  wed 3,
+  thu 4,
+  fri 5,
+  sat 6,
+  sun 7
+);
+$transition-duration: 0.5s;
```

Comment:
Here as well

Reply:
Hi! I can't add space here because Linter is throwing an error( 

src/styles/variables.scss
 9:1  ✖  Unexpected empty line before      scss/dollar-variable-empty-line-before 
         $-variable